### PR TITLE
translate latency

### PR DESCRIPTION
**Pull request by ceci3**
translate latency
fix https://github.com/PaddlePaddle/PaddleSlim/issues/72

### CODE REVIEW

File: docs/en/api_en/table_latency_en.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Table about hardware lantency
+
+The table about hardware latency is used to evaluate the inference time in special environment and forecast. The following text used to introduce the format that PaddleSlim support.
```
Comment (wanghaoshuang) [inline docs/en/api_en/table_latency_en.md]: forecast -> inference engine

File: docs/en/api_en/table_latency_en.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+## Introduce
+
+The table about hardware latency saved all possible operations, one operation in the table including type and parameters, such as: type can be `conv2d`, and corresponding parameters can be the size of feature map, number of kernel, and the size of kernel.
+The latency of every operation depends on hardware and forecast.
+
+## Whole format
```
Comment (wanghaoshuang) [inline docs/en/api_en/table_latency_en.md]: Whole format -> Overview Format

File: docs/en/api_en/table_latency_en.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+The table about hardware latency saved in the way of file or multi-line string.
+The first line of the table about hardware latency saved the information about version, every line in the following represents a operation and its latency.
+
+## Version
+
+The information about version split by comma in the english format, and the detail is hardware, forecast and timestamp.
+
+- ** hardware: ** Used to mark the environment of hardware, including type of architecture, version and so on.
+
+- ** forecast: ** Used to mark forecast, including the name of forecast, version, optimize options and so on.
```
Comment (wanghaoshuang) [inline docs/en/api_en/table_latency_en.md]: forecast -> inference engine

File: docs/en/api_en/table_latency_en.md (hunk 1 above)
Comment (ceci3) [reply docs/en/api_en/table_latency_en.md]: done

File: docs/en/api_en/table_latency_en.md (hunk 2 above)
Comment (ceci3) [reply docs/en/api_en/table_latency_en.md]: done

File: docs/en/api_en/table_latency_en.md (hunk 3 above)
Comment (ceci3) [reply docs/en/api_en/table_latency_en.md]: done

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**